SensorsAnalyticsSDK/src/main/java/com/sensorsdata/analytics/javasdk/SensorsAnalytics.java

```diff
@@ -156,12 +160,18 @@ public BatchConsumer(final String serverUrl, final int bulkSize) {
     }
 
     public BatchConsumer(final String serverUrl, final int bulkSize, final boolean throwException) {
-      this(serverUrl, bulkSize, 0, throwException);
+      this(serverUrl, bulkSize, 0, throwException, null);
     }
 
-    public BatchConsumer(final String serverUrl, final int bulkSize, final int maxCacheSize, final boolean throwException) {
+    public BatchConsumer(final String serverUrl, final int bulkSize,
+         final int maxCacheSize, final boolean throwException) {
```

Comment:
Unnecessary reformat.
```suggestion
    public BatchConsumer(final String serverUrl, final int bulkSize, final int maxCacheSize, final boolean throwException) {
```